src/wasm.h

```diff
@@ -595,6 +595,7 @@ enum StringMeasureOp {
   StringMeasureWTF8,
   StringMeasureWTF16,
   StringMeasureIsUSV,
+  StringMeasureView,
```

Comment:
```suggestion
  StringIsUSVSequence,
  StringViewLengthWTF16,
```
Don't we need to make the op name include it is for wtf16? I understand there's only one `length` op, but I think it's better for readability. Also not sure if we need `Measure` in the name.. `

The above is only my suggestion and I'm not very opinionated, but I think it'd be good to at least include `WTF16` in the last name..

Reply:
I'd prefer to keep the `StringMeasure` prefix since all the ops of instructions have that convention atm. It makes it easy to know which class an op belongs to. But good point on the other thing, being more explicit here might be helpful. How about `StringMeasureViewWTF16` or `StringMeasureWTF16View`? The last one just adds "View" to an existing one, which might be the most consistent.

Reply:
Yeah both sounds good.